### PR TITLE
Make it possible to disable google analytics in altair

### PR DESCRIPTION
**Pull request by jarlehansen**
Added the property `altair.google-analytics.enabled`. When set to false, this will disable google analytics in altair. By default this is set to true.

### CODE REVIEW

File: altair-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/altair/boot/AltairController.java [hunk 1]
```diff
@@ -52,6 +52,9 @@
     @Value("${altair.cdn.version:2.1.4}")
     private String altairCdnVersion;
 
+    @Value("${altair.google-analytics.enabled:true}")
```
Comment (oliemansm) [inline altair-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/altair/boot/AltairController.java]: Could you define this property in https://github.com/graphql-java-kickstart/graphql-spring-boot/blob/master/altair-spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json as well so it's available in intellisense?

Comment (jarlehansen) [reply altair-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/altair/boot/AltairController.java]: Yes, good suggestion. Just added it in the latest commit.

**Comment by jarlehansen** [comment]
Got the project compiling, but there is a problem disabling google analytics. Looking into it

**Comment by imolorhe** [comment]
Yes it has been removed from Altair.

**Pull request closed without merging**